server/src/internalClusterTest/java/org/opensearch/remotemigration/RemoteReplicaRecoveryIT.java

```diff
@@ -89,42 +90,38 @@ public void testReplicaRecovery() throws Exception {
         client().admin()
             .indices()
             .updateSettings(
-                new UpdateSettingsRequest("test").settings(
-                    Settings.builder()
-                        .put(IndexMetadata.SETTING_NUMBER_OF_REPLICAS, 3)
-                        .put("index.routing.allocation.exclude._name", remoteNode)
```

Comment:
why is this exclusion not needed?

Reply:
This was a hack we added earlier as Remote Allocation Deciders were not present to force allocation on remote node.  Now this is no longer needed .